io.github.uowuo.abaddon.yml

```
@@ -0,0 +1,183 @@
+app-id: io.github.uowuo.abaddon
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: abaddon
+finish-args:
+  # Wayland access
+  - --socket=wayland
+  # X11 fallback
+  - --socket=fallback-x11
+  - --share=ipc #x11 depends on this
```

```suggestion
  - --socket=wayland
  - --socket=fallback-x11
  - --share=ipc
```

These comments said "water is wet"